src/main/resources/model/instance.json

```diff
@@ -484,7 +484,12 @@
     "holdingPublicNotes": {
       "type": "search",
       "index": "multilang",
-      "processor": "holdingPublicNotesProcessor"
+      "processor": "holdingsPublicNotesProcessor"
+    },
+    "holdingsPublicNotes": {
```

Comment:
there is no need to duplicate the search fields, old fields can be called with the new values using the 
property `inventorySearchTypes`

working example:
```
    "holdingPublicNotes": {
      "type": "search",
      "index": "multilang",
      "inventorySearchTypes": [ "holdingsPublicNotes" ],
      "processor": "holdingsPublicNotesProcessor"
    }
```

Reply:
Thanks Pavel!